dcs-packaging-tool/dcs-packaging-tool-gui/src/main/java/org/dataconservancy/packaging/gui/view/impl/HeaderViewImpl.java

```diff
@@ -82,10 +84,42 @@ public HeaderViewImpl(Labels labels) {
     }
     
     @Override
-    public void highlightNextPage(int nextPosition) {
-        switch (nextPosition) {
-            case 10:
-            case 11:
+    public void highlightNextPage(Page page) {
+        switch (page) {
+            case HOMEPAGE:
+                // Don't show the location bar in the homepage

```

Comment:
I've run the tool with the header hidden in the home screen. I don't think it look unreasonable. We can address this ourselves later, if other disagrees.